src/components/SharedComponents/Buttons/Button.js

```diff
@@ -20,18 +20,22 @@ const setStyles = ( {
   disabled,
   className
 } ) => {
-  let buttonClass = "rounded-3xl h-13";
+  let buttonClass = "rounded";
+  let textClass = "text-lg text-white font-semibold";
 
   if ( className ) {
     buttonClass += ` ${className}`;
   }
 
   if ( level === "warning" ) {
-    buttonClass += buttonClass.concat( " ", "bg-buttonWarning" );
+    buttonClass += buttonClass.concat( " ", "bg-warningRed" );
   } else if ( level === "primary" ) {
-    buttonClass += buttonClass.concat( " ", "bg-buttonPrimary" );
+    buttonClass += buttonClass.concat( " ", "bg-darkGray" );
+  } else if ( level === "focus" ) {
+    buttonClass += buttonClass.concat( " ", "bg-accessibleGreen" );
```

Comment:
FYI, @abhasinat's updated design used `inatGreen` for the focus button (it was `accessibleGreen` yesterday).